pimd/pim_cmd_common.c

```diff
@@ -444,8 +440,6 @@ void pim_show_membership(struct pim_instance *pim, struct vty *vty, bool uj)
 			}
 		}
 	}
-
-	json_object_free(json);
```

Comment:
There is a memleak here. Consider uj is passed as false. then still json Is malloced at line 362 and is never freed since you have removed it.